docs/installation/production/additional/official_docker.rst

```diff
@@ -0,0 +1,423 @@
+    ADD install.yml ${TETHYSAPP_DIR}/tethysapp-my_first_app/
+
+    RUN /bin/bash -c ". ${CONDA_HOME}/bin/activate tethys \
+      ; cd ${TETHYSAPP_DIR}/tethysapp-my_first_app \
+      ; tethys install -N"
+
+    #########
+    # CHOWN #
+    #########
+    RUN export NGINX_USER=$(grep 'user .*;' /etc/nginx/nginx.conf | awk '{print $2}' | awk -F';' '{print $1}') \
+      ; find ${TETHYSAPP_DIR} ! -user ${NGINX_USER} -print0 | xargs -0 -I{} chown ${NGINX_USER}: {} \
+      ; find ${WORKSPACE_ROOT} ! -user ${NGINX_USER} -print0 | xargs -0 -I{} chown ${NGINX_USER}: {} \
+      ; find ${STATIC_ROOT} ! -user ${NGINX_USER} -print0 | xargs -0 -I{} chown ${NGINX_USER}: {} \
+      ; find ${TETHYS_PERSIST}/keys ! -user ${NGINX_USER} -print0 | xargs -0 -I{} chown ${NGINX_USER}: {} \
+      ; find ${TETHYS_HOME}/tethys ! -user ${NGINX_USER} -print0 | xargs -0 -I{} chown ${NGINX_USER}: {}
+
+
+    #########################
+    # CONFIGURE ENVIRONMENT #
+    #########################
+    EXPOSE 80
+
+
+    ################
+    # COPY IN SALT #
+    ################
+    ADD docker/salt/ /srv/salt/
+
+
+    #######
+    # RUN #
+    #######
+    CMD bash run.sh
+
+Run.sh
+------
+
+The bash script ``run.sh`` is executed during run time to startup and initialize the container. Here is what it's trying to accomplish:
+
+* Create Salt Config.
+* Set extra ENVs to NGINX.
+* Check if Database is ready.
+* Run Salt Scripts to establish the necessary set up for the docker image.
+* Fix permissions.
+* Start supervisor.
+* Showing the logs for supervisor, nginx and tethys.
+
+Run.sh also has these following optional arguments:
+
++---------------------------+------------------------------------------------------------------------------------------+
+| Argument                  | Description                                                                              |
++===========================+==========================================================================================+
+| --background              | run supervisord in background.                                                           |
++---------------------------+------------------------------------------------------------------------------------------+
+| --skip-perm               | skip fixing permissions step.                                                            |
++---------------------------+------------------------------------------------------------------------------------------+
+| --db-max-count            | number of attempt to connect to the database. Default is at 24.                          |
++---------------------------+------------------------------------------------------------------------------------------+
+| --test                    | only run salt scripts.                                                                   |
++---------------------------+------------------------------------------------------------------------------------------+
+
```

Comment:
Add an example of run command overriding the default CMD to show how the run.sh arguments can be used when running the Docker image.

Reply:
Added